soul-plugin/soul-plugin-base/src/main/java/org/dromara/soul/plugin/base/AbstractSoulPlugin.java

```diff
@@ -99,6 +97,14 @@
         return chain.execute(exchange);
     }
 
+    protected Mono<Void> handleSelectorIsNull(final String pluginName, final ServerWebExchange exchange, final SoulPluginChain chain) {
```

Comment:
not used in the AbstractSoulPlugin?

Reply:
old function signature. used as before.

![image](https://user-images.githubusercontent.com/11937539/105631975-fe02e600-5e8b-11eb-97c1-12b948a3eb59.png)